### ISSUE TITLE
Uncaught TypeError: Cannot read property 'slice' of undefined

### ISSUE THREAD

**Issue by grokcodile** [zhuochun/md-writer#64]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.0.5
**System**: Mac OS X 10.11
**Thrown From**: [markdown-writer](https://github.com/zhuochun/md-writer) package, v1.8.1

### Stack Trace

Uncaught TypeError: Cannot read property 'slice' of undefined

```
At /Users/ethan/.atom/packages/markdown-writer/lib/commands.coffee:163

TypeError: Cannot read property 'slice' of undefined
  at Commands._findNextTableCellIdx (/Users/ethan/.atom/packages/markdown-writer/lib/commands.coffee:163:39)
  at Commands.jumpToNextTableCell (/Users/ethan/.atom/packages/markdown-writer/lib/commands.coffee:159:13)
  at Commands.trigger (/Users/ethan/.atom/packages/markdown-writer/lib/commands.coffee:17:5)
  at atom-text-editor.<anonymous> (/Users/ethan/.atom/packages/markdown-writer/lib/main.coffee:134:23)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:241:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at CommandPaletteView.module.exports.CommandPaletteView.confirmed (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/command-palette/lib/command-palette-view.js:159:32)
  at CommandPaletteView.module.exports.SelectListView.confirmSelection (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:338:21)
  at space-pen-li.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:139:19)
  at HTMLOListElement.jQuery.event.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4435:9)
  at HTMLOListElement.elemData.handle (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4121:28)

```

### Commands

```
     -1:50 markdown-writer:insert-new-line (atom-text-editor.editor.is-focused)
     -1:46.6.0 markdown-writer:toggle-italic-text (atom-text-editor.editor.is-focused)
     -1:44.8.0 emmet:insert-formatted-line-break-only (atom-text-editor.editor.is-focused)
     -1:44.8.0 markdown-writer:insert-new-line (atom-text-editor.editor.is-focused)
     -1:44.7.0 emmet:insert-formatted-line-break-only (atom-text-editor.editor.is-focused)
     -1:44.7.0 markdown-writer:insert-new-line (atom-text-editor.editor.is-focused)
     -1:36.9.0 markdown-writer:indent-list-line (atom-text-editor.editor.is-focused)
  3x -1:36.2.0 core:backspace (atom-text-editor.editor.is-focused)
     -1:24.7.0 markdown-writer:insert-link (atom-text-editor.editor.is-focused)
     -1:12.8.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:43.9.0 markdown-writer:insert-link (atom-text-editor.editor.is-focused)
     -0:36.9.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:31.8.0 emmet:insert-formatted-line-break-only (atom-text-editor.editor.is-focused)
     -0:31.8.0 markdown-writer:insert-new-line (atom-text-editor.editor.is-focused)
     -0:12.2.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:02.0 markdown-writer:jump-to-next-table-cell (atom-text-editor.editor)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-material-ui",
      "adventurous-syntax"
    ]
  },
  "markdown-writer": {}
}
```

### Installed Packages

```coffee
# User
adventurous-syntax, v0.2.0
atom-beautify, v0.28.10
atom-material-ui, v0.5.13
autocomplete-ruby, v0.1.0
color-picker, v2.0.11
emmet, v2.3.12
jekyll, v1.0.0
linter-csslint, v1.0.5
linter-rubocop, v0.3.1
linter-scss-lint, v2.0.0
markdown-writer, v1.8.1
minimap, v4.12.2
pigments, v0.9.3

# Dev
No dev packages
```

**Comment by zhuochun** [comment]
Would you help provide the table format you used? So I can re-produce this problem.

**Comment by grokcodile** [comment]
The table was lost when the Atom crashed. I was just testing and really have no idea what I had added to the table. It was something like this:

  kjh  |  lkj |   kj
---|---|---
,m   |  km |  ,m 
  klj | lk  |    kj

However, I can not recreate the condition that caused Atom to crash. I had just installed your markdown writer package along with a few other packages and did not restart Atom. I have had no problems since restarting.
